__tests__/app.test.js

```diff
@@ -144,7 +144,7 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
-describe("POST /api/articles/:article_id/comments", () => {
+describe.only("POST /api/articles/:article_id/comments", () => {
```

Comment:
remove .only's from your tests before pushing please, i now don't know if the tests you have written for PATCH are working as they are not being run before committing